census/tflearn/README.md

```diff
@@ -0,0 +1,114 @@
+# Census: TF.Learn Distributed Training and Prediction
+
+This sample walks you through training and prediction, including distributed training, and hyperparameter tuning.
+
+## Local Training
+
+### Getting Data
+
+Throughout we will use a census dataset for training. For local training, you can read them directly from Google Cloud Storage or if you have an unreliable network, can download them to your local file system.
+
+```
+TRAIN_FILE=gs://tf-ml-workshop/widendeep/adult.data
+EVAL_FILE=gs://tf-ml-workshop/widendeep/adult.test
+```
+
+To use locally simply copy them down with gsutil:
+```
+gsutil cp gs://tf-ml-workshop/widendeep/* data/
+```
+
+And use the local locations whereever the environment variables above are used.
+
+
+### Installing Dependencies
+
+The local training environment for Cloud ML uses your local python installation. So you will need to have Python 2.7 and install TensorFlow following the instructions [here](https://www.tensorflow.org/get_started/os_setup).
+
+### Running Training
+
+Then you can run training with the following command:
+
+```
+gcloud beta ml local train \
+    --module-name trainer.task \
+    --package-path trainer/ \
+    -- \
+    --train-file $TRAIN_FILE \
+    --eval-file $EVAL_FILE \
+    --train-steps 1000 \
+    --job_dir output
+```
+
+You should see very verbose output, ending with a line about saving your model.
+
+To ensure that your model works with Cloud MLs distributed execution environment you can add the `--distributed` flag (before the `--` that seperates user args from CLI args). You will see output from a number of different processes which are communicating through gRPC!
+
+## Run On The Cloud
+
+NOTE: Cloud ML Training Jobs are regional. If you do not want to use the us-central1 region, you will need to copy the training data to a regional bucket that matches the region you wish to choose.
+
+Below we will use `OUTPUT_PATH` to represent the fully qualified GCS location for model checkpoints, summaries, and exports. So for example:
+
+```
+OUTPUT_PATH=gs://<my-bucket>/path/to/my/models/run3
+```
+
```

Comment:
Add instructions to enable the Cloud ML API in the project using https://console.developers.google.com/apis/api/ml.googleapis.com/overview. Mention that it may take a minute or two for it to take effect.